models/user.go

```diff
@@ -65,6 +72,17 @@ func (d *UserData) Value() interface{} {
 		return d.NumericValue
 	case BOOL:
 		return d.BoolValue
+	case MAP:
+		data := &map[string]interface{}{}
+		json.Unmarshal([]byte(d.SerializedValue), data)
+		return data
+	case ARRAY:
```

Comment:
Yeah, was thinking the same, only thing is that I don't know if we can get gorm to understand that :/